zerver/views/webhooks/stripe.py

```diff
@@ -0,0 +1,129 @@
+# Webhooks for external integrations.
+from __future__ import absolute_import
+from django.utils.translation import ugettext as _
+from zerver.lib.actions import check_send_message
+from zerver.lib.response import json_success, json_error
+from zerver.decorator import REQ, has_request_variables, api_key_only_webhook_view
+from zerver.lib.validator import check_dict, check_string
+from zerver.models import Client, UserProfile
+
+from django.http import HttpRequest, HttpResponse
+from six import text_type
+from typing import Dict, Any, Iterable, Optional
+
+from datetime import datetime
+import decimal
+
+from past.utils import old_div
```

Comment:
Right, so this is one way to address the Travis CI error, but another is to just use `//` for integer division, or if doing float division, make it clear you're doing so by e.g. dividing by 100.0, not 100.